dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.csiv2-2.0.feature

```diff
@@ -0,0 +1,14 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
I don't see any reason that this auto feature cannot be changed to io.openliberty.  It is not singleton feature.  It is funny that is is named csiv2 in reality since the auto feature doesn't actually rely on a csiv2 feature being enabled, but rather enterpriseBeansRemote and appSecurity

Reply:
It could be renamed. I only kept it named similar to the original b/c I couldn't rename that one and I wanted them to be paired. If you want me to rename with io.openliberty i can. I believe the CSIV2 is probably indicative of the functionality that is being enabled since remote EJBs required CSIv2.

Reply:
In general the direction is for new features to be io.openliberty if possible.  We are not trying to do renames of existing features.

Reply:
Renamed it `io.openliberty.enterpriseBeansRemote-appSecurity-1.0`